sections/deno.zsh

```diff
@@ -0,0 +1,46 @@
+#
+# Deno
+#
+# Deno is secure runtime for JavaScript and TypeScript.
+# Link: https://deno.land/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_DENO_SHOW="${SPACESHIP_DENO_SHOW=true}"
+SPACESHIP_DENO_PREFIX="${SPACESHIP_DENO_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREFIX"}"
+SPACESHIP_DENO_SUFFIX="${SPACESHIP_DENO_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_DENO_SYMBOL="${SPACESHIP_DENO_SYMBOL="🦕 "}"
+SPACESHIP_DENO_DEFAULT_VERSION="${SPACESHIP_DENO_DEFAULT_VERSION=""}"
+SPACESHIP_DENO_COLOR="${SPACESHIP_DENO_COLOR="cyan"}"
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+# Show current version of Deno
+spaceship_deno() {
+  [[ $SPACESHIP_DENO_SHOW == false ]] && return
+
+  # Show Deno status only for Deno-specific folders
+  [[ -n {mod,dep,main,cli}.ts(#qN^/) ]] || return
+
+  local 'deno_version'
+
+  if spaceship::exists dvm; then
+    deno_version=$(dvm current 2>/dev/null)
+  elif spaceship::exists deno; then
+    deno_version=$(deno --version 2>/dev/null | cut -d' ' -f2)
```

Comment:
```
➜ deno --version
deno 1.0.0-rc2
v8 8.4.300
typescript 3.8.3
```

Reply:
Showing  `1.0.0-rc2 8.4.300 3.8.3` deno, V8, typescript versions together is not expected

Reply:
I'm sorry you are right. I use `dvm` locally so this thing never popped in front of my eyes but yeah, I should've known better. Thank you for noticing!